fdbcli/tests/fdbcli_tests.py

```diff
@@ -334,6 +335,26 @@ def consistencycheck(logger):
     assert output3 == consistency_check_on_output
 
 
+@enable_logging
+def knobmanagment(logger):
```

Comment:
Good set of tests. I think this should also test things like incorrect number of arguments, setting the wrong type for a knob (ie trying to set a double knob to a string), and test `setknob` when not inside a `begin` `commit` block.

Reply:
how do we work around the requirement for having a description if a `setknob` is not inside a  `begin/commit` block? I figured the CLI would just prompt for a description which doesn't work well in automation.

Reply:
Discussed offline that it won't be necessary to test for individual `setknob` (due to prompt for description).